cpp/tests/strings/contains_tests.cpp

```diff
@@ -210,7 +211,7 @@ TEST_F(StringsContainsTests, MatchesIPV4Test)
                                               "5.79.97.178",
                                               "127.0.0.1"});
   auto strings_view = cudf::strings_column_view(strings);
-  {  // is_ip
+  {  // is_ip: 58 instructions
```

Comment:
Is there any way to deduce these numbers without just stepping through the full DFA? It would be nice to have an explanation, but if it's just an empirical observation I guess there's nothing we can do.

Reply:
No, there is no real formula for this. The regex-compiler can produce the number which is what we use for regex-dispatching.